tests/libblight_protocol/test.c

```
@@ -0,0 +1,270 @@
+    assert(errno == EINVAL);
+    blight_message_t message;
+    blight_header_t header;
+    header.type = Repaint;
+    header.ackid = 1;
+    header.size = 0;
+    message.header = header;
+    message.data = NULL;
+    assert(blight_cast_to_surface_info_packet(&message) == NULL);
+    assert(errno == ENODATA);
+    blight_packet_surface_info_t data;
+    message.data = (blight_data_t)&data;
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(uninitvar)